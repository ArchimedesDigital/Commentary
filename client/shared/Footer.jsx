
import FlatButton from 'material-ui/FlatButton';
import baseTheme from 'material-ui/styles/baseThemes/lightBaseTheme';
import getMuiTheme from 'material-ui/styles/getMuiTheme';


Footer = React.createClass({

  getChildContext() {
    return { muiTheme: getMuiTheme(baseTheme) };
  },

  childContextTypes: {
      muiTheme: React.PropTypes.object.isRequired,
  },


  render(){

		let date = new Date();
		let year = date.getFullYear();

    let styles = {
      circleButton : {
        width: "auto",
        height: "auto",
      },
      circleButtonIcon : {
        color: "#ffffff",

      }
    }

    var user_is_loggedin = false;

    return (

      	<footer className="block-shadow">
      		<div className="container">
      			<div className="row footer-nav-row">
      				<div className="col-md-8 col-md-offset-2 col-sm-9 col-sm-offset-1 text-center">
      					<div className="footer-nav-links" role="nav">
      						<FlatButton
                    href="/commentary/"
                    label="Search"
                    linkButton={true}
                    ></FlatButton>
      						<FlatButton
                    href="/commentators/"
                    label="Commentors"
                    linkButton={true}
                    ></FlatButton>
      						<FlatButton
                    href="/topics/"
                    label="Topics"
                    linkButton={true}
                    ></FlatButton>
      						<FlatButton
                    href="/about"
                    label="About"
                    linkButton={true}
                    ></FlatButton>
                  { user_is_loggedin ? "" :
                    <div>
                      <FlatButton
                        href="#"
                        label="Login"
                        linkButton={true}
                        ></FlatButton>
                      <FlatButton
                        href="#"
                        label="Join the Community"
                        linkButton={true}
                        ></FlatButton>
                    </div>
                  }
      					</div>
      				</div>
      			</div>
      			<div className="row mb64 mb-xs-32">
      				<div className="col-md-5 text-right text-left-xs">
      					<h1 className="logo">A Homer Commentary in Progress</h1>
      				</div>

      				<div className="col-md-2 hidden-sm hidden-xs text-center">
      					<a href="http://chs.harvard.edu" target="_blank">
      						<img className="site-logo" src="/images/logo-tower.png"/>
      					</a>
      				</div>

      				<div className="col-md-5 col-sm-6">
      					<p className="lead">
      						For more information about the Commentary or general media inquiries, please contact <a href="mailto:contact@ahcip.chs.harvard.edu">contact@ahcip.chs.harvard.edu</a>.
      					</p>

      					<p className="lead">
      					 This website is provided by <a href="http://chs.harvard.edu" target="_blank">The Center for Hellenic Studies</a>.
      					</p>

      				</div>
      			</div>{/*<!--end of row-->*/}
      			<div className="row">
      				<div className="col-md-8 col-md-offset-2 col-sm-9 col-sm-offset-1 text-center">
      					<p className="fade-1-4 copyright">&copy; 2016 The Center for Hellenic Studies.  See our <a href="/terms" >terms and privacy policy</a>.</p>
      				</div>
      			</div>{/*<!--end of row-->*/}
      		</div>{/*<!--end of container-->*/}
      	</footer>

		);
	}
});
