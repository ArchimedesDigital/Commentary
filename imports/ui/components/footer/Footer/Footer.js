import React from 'react';
import { Meteor } from 'meteor/meteor';
import { createContainer } from 'meteor/react-meteor-data';
import { Session } from 'meteor/session';
import FlatButton from 'material-ui/FlatButton';
import getMuiTheme from 'material-ui/styles/getMuiTheme';

// api
import Settings from '/imports/api/collections/settings';
import Tenants from '/imports/api/collections/tenants';

// lib:
import muiTheme from '/imports/lib/muiTheme';


const Footer = React.createClass({

	propTypes: {
		settings: React.PropTypes.object,
		tenant: React.PropTypes.object,
	},

	childContextTypes: {
		muiTheme: React.PropTypes.object.isRequired,
	},

	getChildContext() {
		return { muiTheme: getMuiTheme(muiTheme) };
	},


	render() {
		const { settings, tenant } = this.props;
		const userIsLoggedin = false;

		return (

			<footer className="block-shadow">
				<div className="container">
					<div className="row footer-nav-row">
						<div className="footer-nav-links" role="navigation">
							{tenant && !tenant.isAnnotation ?
								<div>
									<FlatButton
										href="/commentary"
										label="Commentary"
									/>
									<FlatButton
										href="/commenters"
										label="Commentators"
									/>
									<FlatButton
										href="/keywords"
										label="Words"
									/>
									<FlatButton
										href="/keyideas"
										label="Ideas"
									/>
									<FlatButton
										href="/about"
										label="About"
									/>
								</div>
							: ''}
							{ userIsLoggedin ? '' :
							<div>
								<FlatButton
									href="/sign-in"
									label="Login"
								/>
								<FlatButton
									href="/sign-up"
									label="Join the Community"
								/>
							</div>
							}
						</div>
					</div>
					<div className="row mb64 mb-sm-32">
						<div className="col-md-5 text-right text-left-xs">
							<h1 className="logo">{settings ? settings.name : undefined}</h1>
						</div>

						<div className="col-md-2 hidden-sm hidden-xs text-center">
							<a href="http://chs.harvard.edu" target="_blank" rel="noopener noreferrer">
								<img className="site-logo" src="/images/logo-tower.png" role="presentation" />
							</a>
						</div>

						<div className="col-md-5 col-sm-6 more-info-column">
							<p className="lead">
								For more information about the Commentary or general media inquiries,
								please contact &nbsp;
								<a href={settings && settings.emails ? `mailto:${settings.emails.contact}` : ''}>
									{settings && settings.emails ? settings.emails.contact : ''}
								</a>.
							</p>

							<p className="lead">
								This website is provided by <a href="http://chs.harvard.edu" target="_blank" rel="noopener noreferrer">
									The Center for Hellenic Studies
								</a>.
							</p>

						</div>
					</div>
					{/* <!--end of row-->*/}
					<div className="row">
						<div className="col-md-8 col-md-offset-2 col-sm-9 col-sm-offset-1 text-center">
							<p className="fade-1-4 copyright">
								{settings ? settings.footer : undefined}.
								See our <a href="/terms">terms and privacy policy</a>
							</p>
						</div>
					</div>
					{/* <!--end of row-->*/}
				</div>
				{/* <!--end of container-->*/}
			</footer>

		);
	},
});

export default createContainer(() => {
	const settingsHandle = Meteor.subscribe('settings.tenant', Session.get('tenantId'));

	return {
		settings: settingsHandle.ready() ? Settings.findOne() : {},
		tenant: Tenants.findOne({ _id: Session.get('tenantId') })
	};
}, Footer);
