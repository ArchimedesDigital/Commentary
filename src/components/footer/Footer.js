import React, { Component } from 'react';
import PropTypes from 'prop-types';

import FlatButton from 'material-ui/FlatButton';
import { Link } from 'react-router-dom';
import { compose } from 'react-apollo';

// graphql
import { settingsQuery } from '../../graphql/methods/settings';
import { tenantsQuery } from '../../graphql/methods/tenants';



class Footer extends Component {

	constructor(props) {
		super(props);
		this.state = {};
	}
	componentWillReceiveProps(props) {
		const tenantId = sessionStorage.getItem('tenantId');

		this.setState({
			settings: props.settingsQuery.loading ? {} : props.settingsQuery.settings.find(x => x.tenantId === tenantId),
			tenant: props.tenantsQuery.loading ? undefined : props.tenantsQuery.tenants.find(x => x._id === tenantId)
		});
	}
	render() {
		const { settings, tenant } = this.state;
		const now = new Date();
		const year = now.getFullYear();

		return (

			<footer className="block-shadow">
				<div className="container">
					<div className="row footer-nav-row">
						<div className="footer-nav-links" role="navigation">
							{tenant && !tenant.isAnnotation ?
								<div>
									<Link to="/commentary">
										<FlatButton
											label="Commentary"
										/>
									</Link>
									<Link to="/commenters">
										<FlatButton
											label="Commentators"
										/>
									</Link>
									<Link to="/words">
										<FlatButton
											label="Words"
										/>
									</Link>
									<Link to="/ideas">
										<FlatButton
											label="Ideas"
										/>
									</Link>
									<Link to={settings && settings.aboutURL ? settings.aboutURL : '/about'}>
										<FlatButton
											label="About"
										/>
									</Link>
								</div>
							: ''}
						</div>
					</div>
					<div className="row mb64 mb-sm-32">
						<div className="col-md-5 text-right text-left-xs">
							<h1 className="logo">{settings ? settings.name : undefined}</h1>
						</div>

						<div className="col-md-2 hidden-sm hidden-xs text-center">
							<a href="http://chs.harvard.edu" target="_blank" rel="noopener noreferrer">
								<img
									className="site-logo"
									src="/images/center_for_hellenic_studies_lighthouse.png"
									role="presentation"
									alt="The Center for Hellenic Studies"
								/>
							</a>
						</div>

						<div className="col-md-5 col-sm-6 more-info-column">
							<p className="lead">
								For more information about the Commentary or general media inquiries,
								please contact &nbsp;
								<a href={settings && settings.emails ? `mailto:${settings.emails.contact}` : ''}>
									{settings && settings.emails ? settings.emails.contact : ''}
								</a>.
							</p>

							<p className="lead">
								This website is provided by <a href="http://chs.harvard.edu" target="_blank" rel="noopener noreferrer">
									The Center for Hellenic Studies
								</a>.
							</p>

						</div>
					</div>
					{/* <!--end of row-->*/}
					<div className="row">
						<div className="col-md-8 col-md-offset-2 col-sm-9 col-sm-offset-1 text-center">
							<p className="fade-1-4 copyright">
								Copyright {year} The Center for Hellenic Studies.
								See our <a href="/terms">terms and privacy policy</a>
							</p>
						</div>
					</div>
					{/* <!--end of row-->*/}
				</div>
				{/* <!--end of container-->*/}
			</footer>

		);
	}
}
Footer.propTypes = {
	settingsQuery: PropTypes.object,
	tenantsQuery: PropTypes.object,
};
export default compose(settingsQuery, tenantsQuery)(Footer);
