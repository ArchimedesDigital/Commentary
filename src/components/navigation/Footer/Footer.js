import React, { Component } from 'react';
import PropTypes from 'prop-types';

import FlatButton from 'material-ui/FlatButton';
import { Link } from 'react-router-dom';
import { compose } from 'react-apollo';
import { Grid, Row, Col } from 'react-bootstrap';


// graphql
import settingsQuery from '../../../modules/settings/graphql/queries/list';
import { tenantsQuery } from '../../../graphql/methods/tenants';


import './Footer.css';


class Footer extends Component {

	constructor(props) {
		super(props);
		this.state = {};
	}
	componentWillReceiveProps(props) {
		const tenantId = sessionStorage.getItem('tenantId');

		this.setState({
			settings: props.settingsQuery.loading ? {} : props.settingsQuery.settings.find(x => x.tenantId === tenantId),
			tenant: props.tenantsQuery.loading ? undefined : props.tenantsQuery.tenants.find(x => x._id === tenantId)
		});
	}
	render() {
		const { settings, tenant } = this.state;
		const now = new Date();
		const year = now.getFullYear();

		return (

			<footer className="block-shadow">
				<Grid>
					<Row>
						<div className="footer-nav-row">
							<div className="footer-nav-links" role="navigation">
								{tenant && !tenant.isAnnotation ?
									<div>
										<Link to="/commentary">
											<FlatButton
												label="Commentary"
											/>
										</Link>
										<Link to="/commenters">
											<FlatButton
												label="Commentators"
											/>
										</Link>
										<Link to="/words">
											<FlatButton
												label="Words"
											/>
										</Link>
										<Link to="/ideas">
											<FlatButton
												label="Ideas"
											/>
										</Link>
										<Link to={settings && settings.aboutURL ? settings.aboutURL : '/about'}>
											<FlatButton
												label="About"
											/>
										</Link>
									</div>
									: ''}
							</div>
						</div>
					</Row>
					<Row>
						<Col md={5}>
							<h1 className="logo">{settings ? settings.name : undefined}</h1>
						</Col>

						<Col md={2}>
							<a href="http://chs.harvard.edu" target="_blank" rel="noopener noreferrer">
								<img
									className="site-logo"
									src="/images/center_for_hellenic_studies_lighthouse.png"
									role="presentation"
									alt="The Center for Hellenic Studies"
								/>
							</a>
						</Col>

						<Col md={5}>
							<p className="lead">
								For more information about the Commentary or general media inquiries,
								please contact &nbsp;
								<a href={settings && settings.emails ? `mailto:${settings.emails.contact}` : ''}>
									{settings && settings.emails ? settings.emails.contact : ''}
								</a>.
							</p>

							<p className="lead">
								This website is provided by <a href="http://chs.harvard.edu" target="_blank" rel="noopener noreferrer">
									The Center for Hellenic Studies
								</a>.
							</p>
						</Col>
					</Row>
					<Row>
						<Col md={8}>
							<p className="fade-1-4 copyright">
								Copyright {year} The Center for Hellenic Studies.
								See our <a href="/terms">terms and privacy policy</a>
							</p>
						</Col>
					</Row>
				</Grid>
			</footer>

		);
	}
}
Footer.propTypes = {
	settingsQuery: PropTypes.object,
	tenantsQuery: PropTypes.object,
};
export default compose(settingsQuery, tenantsQuery)(Footer);
