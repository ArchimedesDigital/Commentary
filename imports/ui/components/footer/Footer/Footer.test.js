import React from 'react';
import renderer from 'react-test-renderer';

// component:
import Footer from './Footer';

describe('Footer', () => {
	it('renders correctly', () => {

		const tree = renderer
			.create(<Footer />)
			.toJSON();
		expect(tree).toMatchSnapshot();
	});
});
