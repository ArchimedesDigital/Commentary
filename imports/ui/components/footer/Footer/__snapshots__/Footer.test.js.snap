// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="block-shadow"
>
  <div
    className="container"
  >
    <div
      className="row footer-nav-row"
    >
      <div
        className="footer-nav-links"
        role="navigation"
      >
        <a
          disabled={false}
          href="/commentary"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchStart={[Function]}
          onTouchTap={[Function]}
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
              "backgroundColor": "rgba(0, 0, 0, 0)",
              "border": 10,
              "borderRadius": 2,
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "cursor": "pointer",
              "display": "inline-block",
              "fontFamily": "Roboto, sans-serif",
              "fontSize": "inherit",
              "fontWeight": "inherit",
              "height": 36,
              "lineHeight": "36px",
              "margin": 0,
              "minWidth": 88,
              "muiPrepared": true,
              "outline": "none",
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "textAlign": "center",
              "textDecoration": "none",
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "userSelect": "none",
              "verticalAlign": "middle",
            }
          }
          tabIndex={0}
        >
          <div
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
          >
            <span
              style={
                Object {
                  "fontSize": 14,
                  "fontWeight": 500,
                  "letterSpacing": 0,
                  "muiPrepared": true,
                  "paddingLeft": 16,
                  "paddingRight": 16,
                  "position": "relative",
                  "textTransform": "uppercase",
                  "verticalAlign": "middle",
                }
              }
            >
              Commentary
            </span>
          </div>
        </a>
        <a
          disabled={false}
          href="/commenters"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchStart={[Function]}
          onTouchTap={[Function]}
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
              "backgroundColor": "rgba(0, 0, 0, 0)",
              "border": 10,
              "borderRadius": 2,
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "cursor": "pointer",
              "display": "inline-block",
              "fontFamily": "Roboto, sans-serif",
              "fontSize": "inherit",
              "fontWeight": "inherit",
              "height": 36,
              "lineHeight": "36px",
              "margin": 0,
              "minWidth": 88,
              "muiPrepared": true,
              "outline": "none",
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "textAlign": "center",
              "textDecoration": "none",
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "userSelect": "none",
              "verticalAlign": "middle",
            }
          }
          tabIndex={0}
        >
          <div
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
          >
            <span
              style={
                Object {
                  "fontSize": 14,
                  "fontWeight": 500,
                  "letterSpacing": 0,
                  "muiPrepared": true,
                  "paddingLeft": 16,
                  "paddingRight": 16,
                  "position": "relative",
                  "textTransform": "uppercase",
                  "verticalAlign": "middle",
                }
              }
            >
              Commentators
            </span>
          </div>
        </a>
        <a
          disabled={false}
          href="/keywords"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchStart={[Function]}
          onTouchTap={[Function]}
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
              "backgroundColor": "rgba(0, 0, 0, 0)",
              "border": 10,
              "borderRadius": 2,
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "cursor": "pointer",
              "display": "inline-block",
              "fontFamily": "Roboto, sans-serif",
              "fontSize": "inherit",
              "fontWeight": "inherit",
              "height": 36,
              "lineHeight": "36px",
              "margin": 0,
              "minWidth": 88,
              "muiPrepared": true,
              "outline": "none",
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "textAlign": "center",
              "textDecoration": "none",
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "userSelect": "none",
              "verticalAlign": "middle",
            }
          }
          tabIndex={0}
        >
          <div
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
          >
            <span
              style={
                Object {
                  "fontSize": 14,
                  "fontWeight": 500,
                  "letterSpacing": 0,
                  "muiPrepared": true,
                  "paddingLeft": 16,
                  "paddingRight": 16,
                  "position": "relative",
                  "textTransform": "uppercase",
                  "verticalAlign": "middle",
                }
              }
            >
              Keywords
            </span>
          </div>
        </a>
        <a
          disabled={false}
          href="/keyideas"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchStart={[Function]}
          onTouchTap={[Function]}
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
              "backgroundColor": "rgba(0, 0, 0, 0)",
              "border": 10,
              "borderRadius": 2,
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "cursor": "pointer",
              "display": "inline-block",
              "fontFamily": "Roboto, sans-serif",
              "fontSize": "inherit",
              "fontWeight": "inherit",
              "height": 36,
              "lineHeight": "36px",
              "margin": 0,
              "minWidth": 88,
              "muiPrepared": true,
              "outline": "none",
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "textAlign": "center",
              "textDecoration": "none",
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "userSelect": "none",
              "verticalAlign": "middle",
            }
          }
          tabIndex={0}
        >
          <div
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
          >
            <span
              style={
                Object {
                  "fontSize": 14,
                  "fontWeight": 500,
                  "letterSpacing": 0,
                  "muiPrepared": true,
                  "paddingLeft": 16,
                  "paddingRight": 16,
                  "position": "relative",
                  "textTransform": "uppercase",
                  "verticalAlign": "middle",
                }
              }
            >
              Key Ideas
            </span>
          </div>
        </a>
        <a
          disabled={false}
          href="/about"
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseEnter={[Function]}
          onMouseLeave={[Function]}
          onTouchStart={[Function]}
          onTouchTap={[Function]}
          style={
            Object {
              "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
              "backgroundColor": "rgba(0, 0, 0, 0)",
              "border": 10,
              "borderRadius": 2,
              "boxSizing": "border-box",
              "color": "rgba(0, 0, 0, 0.87)",
              "cursor": "pointer",
              "display": "inline-block",
              "fontFamily": "Roboto, sans-serif",
              "fontSize": "inherit",
              "fontWeight": "inherit",
              "height": 36,
              "lineHeight": "36px",
              "margin": 0,
              "minWidth": 88,
              "muiPrepared": true,
              "outline": "none",
              "overflow": "hidden",
              "padding": 0,
              "position": "relative",
              "textAlign": "center",
              "textDecoration": "none",
              "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
              "userSelect": "none",
              "verticalAlign": "middle",
            }
          }
          tabIndex={0}
        >
          <div
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchStart={[Function]}
          >
            <span
              style={
                Object {
                  "fontSize": 14,
                  "fontWeight": 500,
                  "letterSpacing": 0,
                  "muiPrepared": true,
                  "paddingLeft": 16,
                  "paddingRight": 16,
                  "position": "relative",
                  "textTransform": "uppercase",
                  "verticalAlign": "middle",
                }
              }
            >
              About
            </span>
          </div>
        </a>
        <div>
          <a
            disabled={false}
            href="/sign-in"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "backgroundColor": "rgba(0, 0, 0, 0)",
                "border": 10,
                "borderRadius": 2,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": "inherit",
                "fontWeight": "inherit",
                "height": 36,
                "lineHeight": "36px",
                "margin": 0,
                "minWidth": 88,
                "muiPrepared": true,
                "outline": "none",
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "textAlign": "center",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "verticalAlign": "middle",
              }
            }
            tabIndex={0}
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
            >
              <span
                style={
                  Object {
                    "fontSize": 14,
                    "fontWeight": 500,
                    "letterSpacing": 0,
                    "muiPrepared": true,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "verticalAlign": "middle",
                  }
                }
              >
                Login
              </span>
            </div>
          </a>
          <a
            disabled={false}
            href="/sign-up"
            onBlur={[Function]}
            onClick={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseEnter={[Function]}
            onMouseLeave={[Function]}
            onTouchStart={[Function]}
            onTouchTap={[Function]}
            style={
              Object {
                "WebkitTapHighlightColor": "rgba(0, 0, 0, 0)",
                "backgroundColor": "rgba(0, 0, 0, 0)",
                "border": 10,
                "borderRadius": 2,
                "boxSizing": "border-box",
                "color": "rgba(0, 0, 0, 0.87)",
                "cursor": "pointer",
                "display": "inline-block",
                "fontFamily": "Roboto, sans-serif",
                "fontSize": "inherit",
                "fontWeight": "inherit",
                "height": 36,
                "lineHeight": "36px",
                "margin": 0,
                "minWidth": 88,
                "muiPrepared": true,
                "outline": "none",
                "overflow": "hidden",
                "padding": 0,
                "position": "relative",
                "textAlign": "center",
                "textDecoration": "none",
                "transition": "all 450ms cubic-bezier(0.23, 1, 0.32, 1) 0ms",
                "userSelect": "none",
                "verticalAlign": "middle",
              }
            }
            tabIndex={0}
          >
            <div
              onMouseDown={[Function]}
              onMouseLeave={[Function]}
              onMouseUp={[Function]}
              onTouchEnd={[Function]}
              onTouchStart={[Function]}
            >
              <span
                style={
                  Object {
                    "fontSize": 14,
                    "fontWeight": 500,
                    "letterSpacing": 0,
                    "muiPrepared": true,
                    "paddingLeft": 16,
                    "paddingRight": 16,
                    "position": "relative",
                    "textTransform": "uppercase",
                    "verticalAlign": "middle",
                  }
                }
              >
                Join the Community
              </span>
            </div>
          </a>
        </div>
      </div>
    </div>
    <div
      className="row mb64 mb-sm-32"
    >
      <div
        className="col-md-5 text-right text-left-xs"
      >
        <h1
          className="logo"
        />
      </div>
      <div
        className="col-md-2 hidden-sm hidden-xs text-center"
      >
        <a
          href="http://chs.harvard.edu"
          rel="noopener noreferrer"
          target="_blank"
        >
          <img
            className="site-logo"
            role="presentation"
            src="/images/logo-tower.png"
          />
        </a>
      </div>
      <div
        className="col-md-5 col-sm-6 more-info-column"
      >
        <p
          className="lead"
        >
          For more information about the Commentary or general media inquiries, please contact  
          <a
            href=""
          >
            
          </a>
          .
        </p>
        <p
          className="lead"
        >
          This website is provided by 
          <a
            href="http://chs.harvard.edu"
            rel="noopener noreferrer"
            target="_blank"
          >
            The Center for Hellenic Studies
          </a>
          .
        </p>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-8 col-md-offset-2 col-sm-9 col-sm-offset-1 text-center"
      >
        <p
          className="fade-1-4 copyright"
        >
          . See our 
          <a
            href="/terms"
          >
            terms and privacy policy
          </a>
        </p>
      </div>
    </div>
  </div>
</footer>
`;
